src/functionBased/components/TodoContainer.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
+import About from '../pages/About';
+import NotMatch from '../pages/NotMatch';
+import Navbar from './Navbar';
+
+const TodoContainer = () => {
+  function getInitialTodos() {
+    // getting stored items
```

Comment:
- [ ] Kindly take out the commented code as this would help put you in the habit of writing cleaner and better code